package.xml

```diff
@@ -11,8 +11,14 @@
   <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
   <build_depend>rosidl_default_generators</build_depend>
+  <!-- TODO(clalancette): This dependency is really wrong; std_msgs is only used
+       as part of the tests.  But it turns out that rosidl_generate_interfaces()
+       always adds it to the exported dependencies, so we need to export this
+       incorrect dependency for now
```

Comment:
> I'm confused by this comment. I would think listing `std_msgs` as a `test_depend` is enough. Does declaring it as a `build_depend` cause it to be exported?

Basically, when I made this a `test_depend` only, CI failed because it couldn't find `std_msgs`.  But thinking about it more, I'm honestly not sure why that is the case; the `test_depend` should have been enough.

Since this is a side issue for this PR, I'm actually going to get rid of that comment and instead open a new issue here so we can address this dependency separately.